snipssonos/use_cases/request_objects.py

```diff
@@ -265,51 +265,50 @@ class PlayMusicRequestFactory(RequestObjectFactory):
 class InjectEntitiesRequestObject(ValidRequestObject):
     VALID_ENTITY_NAMES = ["artists", "tracks", "playlists"]
 
-    def __init__(self, entity_name, entity_slot_name):
-        self.entity_name = entity_name
-        self.entity_slot_name = entity_slot_name
+    def __init__(self, entities):
+        self.entities = entities
 
     @property
-    def entity_name(self):
-        return self._entity_name
+    def entities(self):
+        return self._entities
 
-    @property
-    def entity_slot_name(self):
-        return self._entity_slot_name
-
-    @entity_name.setter
-    def entity_name(self, entity_name):
+    @entities.setter
+    def entities(self, entities):
```

Comment:
I think the validation of properties is a bit overzealous here. We are not doing validation here, we are doing type checking. 
Because Python is dynamically typed, we end up with too much code. 
A Python3 annotation would do the job imho (in this case, a well written comment).

Reply:
We are doing validation and type checking as seen in the other request objects. A part from ensuring that the data in the object is the right type we are also ensuring that the entity names used as well as the entity slot names (to be done once are set in stone) are, indeed, valid.